Editor/UI/AnalysisView.cs

```diff
@@ -192,23 +192,30 @@ public void OnGUI()
         void DrawTable(ProjectIssue[] selectedIssues)
         {
             EditorGUILayout.BeginVertical();
+            EditorGUILayout.Space();
 
-            var r = EditorGUILayout.GetControlRect(GUILayout.ExpandHeight(true));
-
-            Profiler.BeginSample("IssueTable.OnGUI");
-            m_Table.OnGUI(r);
-            Profiler.EndSample();
-
+            EditorGUILayout.BeginHorizontal();
             if (m_Desc.groupByDescription)
             {
-                EditorGUILayout.BeginHorizontal();
-                if (GUILayout.Button(Styles.CollapseAllButton, GUILayout.ExpandWidth(true), GUILayout.Width(100)))
+                if (GUILayout.Button(Styles.CollapseAllButton, EditorStyles.toolbarButton, GUILayout.ExpandWidth(true), GUILayout.Width(100)))
                     SetRowsExpanded(false);
-                if (GUILayout.Button(Styles.ExpandAllButton, GUILayout.ExpandWidth(true), GUILayout.Width(100)))
+                if (GUILayout.Button(Styles.ExpandAllButton, EditorStyles.toolbarButton, GUILayout.ExpandWidth(true), GUILayout.Width(100)))
                     SetRowsExpanded(true);
-                EditorGUILayout.EndHorizontal();
             }
 
+            EditorGUILayout.Space();
+            EditorGUILayout.LabelField("Zoom: ", EditorStyles.toolbarButton, GUILayout.ExpandWidth(false), GUILayout.Width(80));
+            m_Preferences.fontSize = (int)GUILayout.HorizontalSlider(m_Preferences.fontSize, Preferences.k_MinFontSize, Preferences.k_MaxFontSize, GUILayout.ExpandWidth(false), GUILayout.Width(100));
+            m_Table.SetFontSize(m_Preferences.fontSize);
```

Comment:
Another option would be to store 2 `GUIStyle`s (textarea and label) in `Preferences.cs` and update their attributes in a setter for `m_Preferences.fontSize`. It would avoid recreating them all the time too (not sure that really matters though)